### PR TITLE
Updating screenshots and markdown of "Invoke an API Using a SOAP Client"

### PR DESCRIPTION
**Pull request by rashm1n**
## Purpose

$subject

## Updated Screenshots

generate-keys-phone-verification
oauth2-phone-verification
subscribe-phone-verification

### CODE REVIEW

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md [hunk 1]
```diff
@@ -11,24 +11,36 @@ Let's invoke the `PhoneVerification` API using a SOAP client.
...
-
-    ![]({{base_path}}/assets/img/learn/generate-keys-production-default.png)
-
-4.  Copy the access token to the clipboard as you need it later to invoke the API.
-
-5.  Download the SOAP UI installation that suits your operating system from <https://www.soapui.org/downloads/soapui.html> and open its console.
-6.  In the SOAP UI, right click on the **Projects** menu and create a new SOAP project.
+3.  Click the **Applications** menu, open the default application using which you subscribed to the API, and navigate to **OAuth2 Tokens** under **Production Keys**.
+    
+    ![]({{base_path}}/assets/img/learn/oauth2-phone-verification.png)
```
Comment (Mariangela) [inline en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md:16]:
```suggestion
    [![oauth2 phone verification]({{base_path}}/assets/img/learn/oauth2-phone-verification.png)]({{base_path}}/assets/img/learn/oauth2-phone-verification.png)
```

Comment (Mariangela) [inline en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md:16]: As suggested above pls add ALT tags and make the image link zoomable

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md [hunk 2]
```diff
@@ -43,9 +55,10 @@ Let's invoke the `PhoneVerification` API using a SOAP client.
         ![]({{base_path}}/assets/attachments/103332601/103332589.png)
 
     Note the result on the right-hand side panel. As you gave a dummy phone number in this example, you get the result as invalid.
-    ![]({{base_path}}/assets/attachments/103332601/103332590.png)
+    
+![]({{base_path}}/assets/attachments/103332601/103332590.png)
```
Comment (Mariangela) [inline en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md:59]: If you come across images or attachments with numbers like this please rename them using a meaningful name.

Comment (rashm1n) [reply en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md]: Ack

File: en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md (hunk 1 above)
Comment (rashm1n) [reply en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md:16]: Ack

**Review by Mariangela — APPROVED** [review]

**Pull request merged**